tools/src/main/scala/scala/scalanative/linker/ClassPath.scala

```diff
@@ -20,14 +20,16 @@ sealed trait ClassPath {
 object ClassPath {
 
   /** Create classpath based on the directory. */
-  def apply(directory: Path): ClassPath =
-    new Impl(VirtualDirectory.local(directory))
+  def apply(directory: Path, config: build.Config): ClassPath =
+    new Impl(VirtualDirectory.local(directory), config)
```

Comment:
It looks like config is not used any more. It was probably added previously to support previous version of sizeof that desugared in binary serializer.